<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>rmelting • rmelting</title>
<!-- jquery --><script src="https://code.jquery.com/jquery-3.1.0.min.js" integrity="sha384-nrOSfDHtoPMzJHjVTdCopGqIqeYETSXhZDFyniQ8ZHcVy08QesyHcnOUpMpqnmWq" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script><!-- Font Awesome icons --><link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-T8Gy5hrqNKT+hzMclPo118YTQO6cYprQmhrYwIiQ/3axmI1hQomh7Ud2hPOy8SP1" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/1.7.1/clipboard.min.js" integrity="sha384-cV+rhyOuRHc9Ub/91rihWcGmMmCXDeksTtCihMupQHSsi8GIIRDG0ThDc3HGQFJ3" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../jquery.sticky-kit.min.js"></script><script src="../pkgdown.js"></script><meta property="og:title" content="rmelting">
<meta property="og:description" content="">
<meta name="twitter:card" content="summary">
<!-- mathjax --><script src="https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <div class="container template-vignette">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">rmelting</a>
        <span class="label label-default" data-toggle="tooltip" data-placement="bottom" title="Released package">0.0.0.9000</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fa fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/doc.html">rmelting</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/aravind-j/rmelting">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      
      </header><div class="row">
  <div class="col-md-9">
    <div class="page-header toc-ignore">
      <h1>rmelting</h1>
                        <h4 class="author">Temp</h4>
            
            <h4 class="date">3 April 2018</h4>
      
      <small>Source: <a href="https://github.com/aravind-j/rmelting/blob/master/vignettes/doc.Rmd"><code>vignettes/doc.Rmd</code></a></small>

    </div>

    
    
<div class="contents">
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1"><span class="kw">library</span>(printr)</a>
<a class="sourceLine" id="cb1-2" data-line-number="2"><span class="kw">library</span>(rmelting)</a></code></pre></div>
<pre><code>## Loading required package: rJava</code></pre>
<pre><code>## 
## --------------------------------------------------------------------------------
## Welcome to rmelting version 0.1.0
## 
## 
## # To know whats new in this version type:
##   news(package='rmelting')
##   for the NEWS file.
## 
## # To cite the methods in the package type:
##   citation(package='rmelting')
## 
## # To suppress this message use:
##   suppressPackageStartupMessages(library(rmelting))
## --------------------------------------------------------------------------------</code></pre>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb4-1" data-line-number="1">?melting</a></code></pre></div>
<div class="r-help-page">
<table width="100%" summary="page for melting" class="table"><tr>
<td>
melting
</td>
<td style="text-align: right;">
R Documentation
</td>
</tr></table>
<h2>
melting
</h2>
<h3>
Description
</h3>
<p>
R interface to the MELTING 5 software for computation of enthalpy and entropy of the helix-coil transition, and then the melting temperature of a nucleic acid duplex.
</p>
<h3>
Usage
</h3>
<pre class="r">
melting(sequence, comp.sequence = NULL, nucleic.acid.conc,
  hybridisation.type = c("dnadna", "rnarna", "dnarna", "rnadna", "mrnarna",
  "rnamrna"), Na.conc, Mg.conc, Tris.conc, K.conc, dNTP.conc, DMSO.conc,
  formamide.conc, size.threshold = 60, self = FALSE, correction.factor,
  method.approx = c("ahs01", "che93", "che93corr", "marschdot", "owe69",
  "san98", "wetdna91", "wetrna91", "wetdnarna91"), method.nn = c("all97",
  "bre86", "san04", "san96", "sug96", "tan04", "fre86", "xia98", "sug95",
  "tur06"), method.GU = c("tur99"), method.singleMM, method.tandemMM,
  method.single.dangle, method.double.dangle, method.long.dangle,
  method.internal.loop, method.single.bulge.loop, method.long.bulge.loop,
  method.CNG, method.inosine, method.hydroxyadenine, method.azobenzenes,
  method.locked, correction.Na, correction.Mg, correction.NaMg, method.Naeq,
  correction.DMSO, correction.formamide)
</pre>
<h3>
Arguments
</h3>
<table summary="R argblock" class="table">
<tr valign="top">
<td>
<code>sequence</code>
</td>
<td>
<p>
Sequence (5’ to 3’) of one strand of the nucleic acid duplex as a character string.
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>comp.sequence</code>
</td>
<td>
<p>
Complementary sequence (3’ to 5’) of the nucleic acid duplex as a character string.
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>nucleic.acid.conc</code>
</td>
<td>
<p>
Concentration of the nucleic acid strand (<i>mol/L</i>) in excess as a numeric value.
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>hybridisation.type</code>
</td>
<td>
<p>
The hybridisation type. Either <code>“dnadna”</code>, <code>“rnarna”</code>, <code>“dnarna”</code>, <code>“rnadna”</code>, <code>“mrnarna”</code> or <code>“rnamrna”</code> (see <strong>Hybridisation type options</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>Na.conc</code>
</td>
<td>
<p>
Concentration of Na ions (M) as a numeric value (see <strong>Ion and agent concentrations</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>Mg.conc</code>
</td>
<td>
<p>
Concentration of Mg ions (M) as a numeric value (see <strong>Ion and agent concentrations</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>Tris.conc</code>
</td>
<td>
<p>
Concentration of Tris ions (M) as a numeric value (see <strong>Ion and agent concentrations</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>K.conc</code>
</td>
<td>
<p>
Concentration of K ions (M) as a numeric value (see <strong>Ion and agent concentrations</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>dNTP.conc</code>
</td>
<td>
<p>
Concentration of dNTP (M) as a numeric value (see <strong>Ion and agent concentrations</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>DMSO.conc</code>
</td>
<td>
<p>
Concentration of DMSO (%) as a numeric value (see <strong>Ion and agent concentrations</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>formamide.conc</code>
</td>
<td>
<p>
Concentration of formamide (M or % depending on correction method) as a numeric value (see <strong>Ion and agent concentrations</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>size.threshold</code>
</td>
<td>
<p>
Sequence length threshold to decide approximative or nearest-neighbour approach for computation. Default is 60.
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>self</code>
</td>
<td>
<p>
logical. Specifies that <code>sequence</code> is self complementary and <code>complementary sequence</code> is not required (seed <strong>Self complementary sequences</strong>). Default is <code>FALSE</code>.
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>correction.factor</code>
</td>
<td>
<p>
Correction factor to be used to modulate the effect of the nucleic acid concentration (<code>nucleic.acid.conc</code>) in the computation of melting temperature (see <strong>Correction factor for nucleic acid concentration</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>method.approx</code>
</td>
<td>
<p>
Specify the approximative formula to be used for melting temperature calculation for sequences of length greater than <code>size.threshold</code>. Either <code>“ahs01”</code>, <code>“che93”</code>, <code>“che93corr”</code>, <code>“schdot”</code>, <code>“owe69”</code>, <code>“san98”</code>, <code>“wetdna91”</code>, <code>“wetrna91”</code> or <code>“wetdnarna91”</code> (see <strong>Approximative formulas</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>method.nn</code>
</td>
<td>
<p>
Specify the nearest neighbor model to be used for melting temperature calculation for sequences of length lesser than <code>size.threshold</code>. Either <code>“all97”</code>, <code>“bre86”</code>, <code>“san04”</code>, <code>“san96”</code>, <code>“sug96”</code>, <code>“tan04”</code>, <code>“fre86”</code>, <code>“xia98”</code>, <code>“sug95”</code> or <code>“tur06”</code> (see <strong>Nearest neighbor models</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>method.GU</code>
</td>
<td>
<p>
Specify the nearest neighbor model to compute the contribution of GU base pairs to the thermodynamic of helix-coil transition. Available method is <code>“tur99”</code> (see <strong>GU wobble base pairs effect</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>method.singleMM</code>
</td>
<td>
<p>
Specify the nearest neighbor model to compute the contribution of single mismatch to the thermodynamic of helix-coil transition. Either <code>“allsanpey”</code>, <code>“tur06”</code>, <code>“zno07”</code> or <code>“zno08”</code> (see <strong>Single mismatch effect</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>method.tandemMM</code>
</td>
<td>
<p>
Specify the nearest neighbor model to compute the contribution of tandem mismatches to the thermodynamic of helix-coil transition. Either <code>“allsanpey”</code> or <code>“tur99”</code> (see <strong>Tandem mismatches effect</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>method.single.dangle</code>
</td>
<td>
<p>
Specify the nearest neighbor model to compute the contribution of single dangling end to the thermodynamic of helix-coil transition. Either <code>“bom00”</code>, <code>“sugdna02”</code>, <code>“sugrna02”</code> or <code>“ser08”</code> (see <strong>Single dangling end effect</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>method.double.dangle</code>
</td>
<td>
<p>
Specify the nearest neighbor model to compute the contribution of double dangling end to the thermodynamic of helix-coil transition. Either <code>“sugdna02”</code>, <code>“sugrna02”</code>, <code>“ser05”</code> or <code>“ser06”</code> (see <strong>Double dangling end effect</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>method.long.dangle</code>
</td>
<td>
<p>
Specify the nearest neighbor model to compute the contribution of long dangling end to the thermodynamic of helix-coil transition. Either <code>“sugdna02”</code> or <code>“sugrna02”</code> (see <strong>Long dangling end effect</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>method.internal.loop</code>
</td>
<td>
<p>
Specify the nearest neighbor model to compute the contribution of internal loop to the thermodynamic of helix-coil transition. Either <code>“san04”</code>, <code>“tur06”</code> or <code>“zno07”</code> (see <strong>Internal loop effect</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>method.single.bulge.loop</code>
</td>
<td>
<p>
Specify the nearest neighbor model to compute the contribution of single bulge loop to the thermodynamic of helix-coil transition. Either <code>“san04”</code>, <code>“tan04”</code>, <code>“ser07”</code> or <code>“tur06”</code> (see <strong>Single bulge loop effect</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>method.long.bulge.loop</code>
</td>
<td>
<p>
Specify the nearest neighbor model to compute the contribution of long bulge loop to the thermodynamic of helix-coil transition. Either <code>“san04”</code> or <code>“tur06”</code> (see <strong>Long bulge loop effect</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>method.CNG</code>
</td>
<td>
<p>
Specify the nearest neighbor model to compute the contribution of CNG repeats to the thermodynamic of helix-coil transition. Available method is <code>“bro05”</code> (see <strong>CNG repeats effect</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>method.inosine</code>
</td>
<td>
<p>
Specify the pecific nearest neighbor model to compute the contribution of inosine bases (I) to the thermodynamic of helix-coil transition. Either <code>“san05”</code> or <code>“zno07”</code> (see <strong>Inosine bases effect</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>method.hydroxyadenine</code>
</td>
<td>
<p>
Specify the nearest neighbor model to compute the contribution of hydroxyadenine bases (A*) to the thermodynamic of helix-coil transition. Available method is <code>“sug01”</code> (see <strong>Hydroxyadenine bases effect</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>method.azobenzenes</code>
</td>
<td>
<p>
Specify the nearest neighbor model to compute the contribution of azobenzenes (X_T for trans azobenzenes and X_C for cis azobenzenes) to the thermodynamic of helix-coil transition. Available method is <code>“asa05”</code> (see <strong>Azobenzenes effect</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>method.locked</code>
</td>
<td>
<p>
Specify the nearest neighbor model to compute the contribution of locked nucleic acids (AL, GL, TL and CL) to the thermodynamic of helix-coil transition. Available method is <code>“mct04”</code> (see <strong>Locked nucleic acids effect</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>correction.Na</code>
</td>
<td>
<p>
Specify the correction method for Na ions. Either <code>“ahs01”</code>, <code>“kam71”</code>, <code>“owc1904”</code>, <code>“owc2004”</code>, <code>“owc2104”</code>, <code>“owc2204”</code>, <code>“san96”</code>, <code>“san04”</code>, <code>“schlif”</code>, <code>“tanna06”</code>, <code>“wetdna91”</code>, <code>“tanna07”</code>, <code>“wetrna91”</code> or <code>“wetdnarna91”</code> (see <strong>Sodium corrections</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>correction.Mg</code>
</td>
<td>
<p>
Specify the correction method for Mg ions. Either <code>“owcmg08”</code>, <code>“tanmg06”</code> or <code>“tanmg07”</code> (see <strong>Magnesium corrections</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>correction.NaMg</code>
</td>
<td>
<p>
Specify the correction method for mixed Na and Mg ions. Either <code>“owcmix08”</code>, <code>“tanmix07”</code> or <code>“tanmix07”</code> (see <strong>Mixed Sodium and Magnesium corrections</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>method.Naeq</code>
</td>
<td>
<p>
Specify the ion correction which gives a sodium equivalent concentration if other cations are present. Either <code>“ahs01”</code>, <code>“mit96”</code> or <code>“pey00”</code> (see <strong>Sodium equivalent concentration methods</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>correction.DMSO</code>
</td>
<td>
<p>
Specify the correction method for DMSO. Specify the correction method for DMSO. Either <code>“ahs01”</code>, <code>“mus81”</code>, <code>“cul76”</code> or <code>“esc80”</code> (see <strong>DMSO corrections</strong>).
</p>
</td>
</tr>
<tr valign="top">
<td>
<code>correction.formamide</code>
</td>
<td>
<p>
Specify the correction method for formamide. Specify the correction method for formamide Either <code>“bla96”</code> or <code>“lincorr”</code> (see <strong>Formamide corrections</strong>).
</p>
</td>
</tr>
</table>
<h3>
Mandatory arguments
</h3>
<p>
The following are the arguments which are mandatory for computation.
</p>
<ul>
<li>
<p>
<code>sequence</code>
</p>
</li>
<li>
<p>
<code>comp.sequence</code>: Mandatory if there are mismatches, inosine(s) or hydroxyadenine(s) between the two strands. If not specified, it is computed as the complement of <code>sequence</code>. Self-complementarity in <code>sequence</code> is detected even though there may be (are) dangling end(s) and <code>comp.sequence</code> is computed (see <strong>Self complementary sequences</strong>).
</p>
</li>
<li>
<p>
<code>nucleic.acid.conc</code>
</p>
</li>
<li>
<p>
<code>Na.conc, Mg.conc, Tris.conc, K.conc</code>: At least one cation (Na, Mg, Tris, K) concentration is mandatory, the other agents(dNTP, DMSO, formamide) are optional.
</p>
</li>
<li>
<p>
<code>hybridisation.type</code>
</p>
</li>
</ul>
<h3>
Hybridisation type options
</h3>
<p>
The details of the possible options for hybridisation type specified in the argument <code>hybridisation.type</code> are as follows:
</p>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Option</strong>
</td>
<td style="text-align: left;">
<strong><code>Sequence</code></strong>
</td>
<td style="text-align: left;">
<strong><code>Complementary sequence</code></strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>dnadna</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
DNA
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>rnarna</code>
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
RNA
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>dnarna</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
RNA
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>rnadna</code>
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
DNA
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>mrnarna</code>
</td>
<td style="text-align: left;">
2-o-methyl RNA
</td>
<td style="text-align: left;">
RNA
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>rnamrna</code>
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
2-o-methyl RNA
</td>
</tr>
</table>
<p>
This parameter determines the nature of the sequences in the arguments <code>sequence</code> and <code>comp.sequence</code>.
</p>
<h3>
Ion and agent concentrations
</h3>
<p>
These values are used for different correction functions which approximately adjusts for effects of these ions(Na, Mg, Tris, K) and/or agents(dNTP, DMSO, formamide) on on thermodynamic stability of nucleic acid duplexes. Their concentration limits depends on the correction method used. All the concentrations must be in M, except for the DMSO (%) and formamide (% or M depending on the correction method). Note that Tris+ concentration is about half of the total tris buffer concentration.
</p>
<h3>
Self complementary sequences
</h3>
<p>
Self complementarity for perfect matching sequences or sequences with dangling ends is detected automatically. However it can be specified by the argument <code>self</code>.
</p>
<h3>
Correction factor for nucleic acid concentration
</h3>
<p>
For self complementary sequences (Auto detected or specified in <code>self</code>) it is 1. Otherwise it is 4 if the both strands are present in equivalent amount and 1 if one strand is in excess.
</p>
<h3>
Approximative estimation formulas
</h3>
<p>
The calculation is increasingly incorrect when the length of the duplex decreases. Moreover, it does not take into account nucleic acid concentration.
</p>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Formula</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>ahs01</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
No mismatch
</td>
<td style="text-align: left;">
von Ahsen, Wittwer, and SchÃ¼tz 2001
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>che93</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
No mismatch; Na=0, Mg=0.0015, Tris=0.01, K=0.05
</td>
<td style="text-align: left;">
Marmur and Doty 1962
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>che93corr</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
No mismatch; Na=0, Mg=0.0015, Tris=0.01, K=0.05
</td>
<td style="text-align: left;">
Marmur et al., 1962
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>marschdot</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
No mismatch
</td>
<td style="text-align: left;">
Wetmur 1991; Marmur et al., 1962; Chester and Marshak 1993; Schildkraut and Lifson 1965; Wahl, Barger, and Kimmel 1987; Britten, Graham, and Neufeld 1974; Hall, Grula, Davidson, and Britten 1980
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>owe69</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
No mismatch
</td>
<td style="text-align: left;">
Owen, Hill, and Lapage 1969; Frank-Kamenetskii 1971; Blake 1996; Blake and Delcourt 1998
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>san98</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
No mismatch
</td>
<td style="text-align: left;">
SantaLucia 1998; von Ahsen et al., 2001
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>wetdna91</code>*
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
Wetmur 1991
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>wetrna91</code>*
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
Wetmur 1991
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>wetdnarna91</code>*
</td>
<td style="text-align: left;">
DNA/RNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
Wetmur 1991
</td>
</tr>
</table>
<h3>
Nearest neighbor models
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Model</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>all97</code>*
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
Allawi et al., 1997
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>bre86</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
Breslauer et al., 1986
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>san04</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
SantaLucia et al., 2004
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>san96</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
SantaLucia et al., 1996
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>sug96</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
Sugimoto et al., 1996
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>tan04</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
Tanaka et al., 2004
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>fre86</code>
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
Freier et al., 1986
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>xia98</code>*
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
Xia et al., 1998
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>sug95</code>*
</td>
<td style="text-align: left;">
DNA/RNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
SantaLucia et al., 1996
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>tur06</code>*
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
A sodium correction (<code>san04</code>) is automatically applied to convert the entropy (Na = 0.1M) into the entropy (Na = 1M)
</td>
<td style="text-align: left;">
Kierzek et al., 2006
</td>
</tr>
</table>
<h3>
GU wobble base pairs effect
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Model</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>tur99</code>*
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
Mathews et al., 1999
</td>
</tr>
</table>
<h3>
Single mismatch effect
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Model</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>allsanpey</code>*
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
Allawi et al., 1997; Allawi et al., 1998a; Allawi et al., 1998b; Allawi et al., 1998; Peyret et al., 1999
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>tur06</code>
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
Lu et al., 2006
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>zno07</code>*
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
Davis et al., 2007
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>zno08</code>
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
At least one adjacent GU base pair.
</td>
<td style="text-align: left;">
Davis et al., 2008
</td>
</tr>
</table>
<h3>
Tandem mismatches effect
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Model</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>allsanpey</code>*
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Only GT mismatches and TA/TG mismatches.
</td>
<td style="text-align: left;">
Allawi et al., 1997; Allawi et al., 1998a; Allawi et al., 1998b; Allawi et al., 1998; Peyret et al., 1999
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>tur99</code>*
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
No adjacent GU or UG base pairs.
</td>
<td style="text-align: left;">
Mathews et al., 1999
</td>
</tr>
</table>
<p>
Tandem mismatches are not taken into account by the approximative mode. Note that not all the mismatched Crick’s pairs have been investigated.
</p>
<h3>
Single dangling end effect
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Model</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>bom00</code>*
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
NA
</td>
<td style="text-align: left;">
Bommarito et al., 2000
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>sugdna02</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Only terminal poly A self complementary sequences.
</td>
<td style="text-align: left;">
Ohmichi et al., 2002
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>sugrna02</code>
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
Only terminal poly A self complementary sequences.
</td>
<td style="text-align: left;">
Ohmichi et al., 2002
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>ser08</code>*
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
Only 3’ UA, GU and UG terminal base pairs only 5’ UG and GU terminal base pairs.
</td>
<td style="text-align: left;">
Miller et al., 2008
</td>
</tr>
</table>
<p>
Single dangling ends are not taken into account by the approximative mode.
</p>
<h3>
Double dangling end effect
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Model</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>sugdna02</code>*
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Only terminal poly A self complementary sequences.
</td>
<td style="text-align: left;">
Ohmichi et al., 2002
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>sugrna02</code>
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
Only terminal poly A self complementary sequences.
</td>
<td style="text-align: left;">
Ohmichi et al., 2002
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>ser05</code>
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
Depends on the available thermodynamic parameters for single dangling end.
</td>
<td style="text-align: left;">
O’Toole et al., 2005
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>ser06</code>*
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
O’Toole et al., 2006
</td>
</tr>
</table>
<p>
Double dangling ends are not taken into account by the approximative mode.
</p>
<h3>
Long dangling end effect
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Model</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>sugdna02</code>*
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Only terminal poly A self complementary sequences.
</td>
<td style="text-align: left;">
Ohmichi et al., 2002
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>sugrna02</code>*
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
Only terminal poly A self complementary sequences.
</td>
<td style="text-align: left;">
Ohmichi et al., 2002
</td>
</tr>
</table>
<p>
Long dangling ends are not taken into account by the approximative mode.
</p>
<h3>
Internal loop effect
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Model</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>san04</code>*
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Missing asymmetry penalty. Not tested with experimental results.
</td>
<td style="text-align: left;">
SantaLucia and Hicks 2004
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>tur06</code>
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
Not tested with experimental results.
</td>
<td style="text-align: left;">
Lu et al., 2006
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>zno07</code>*
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
Davis et al., 2007
</td>
</tr>
</table>
<p>
Internal loops are not taken into account by the approximative mode.
</p>
<h3>
Single bulge loop effect
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Model</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>tan04</code>*
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
Tan and Chen 2007
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>san04</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Missing closing AT penalty.
</td>
<td style="text-align: left;">
SantaLucia et al., 2004
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>ser07</code>
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
Less reliable results. Some missing parameters.
</td>
<td style="text-align: left;">
Blose, Manni, Klapec, Stranger-Jones, Zyra, Sim, Griffith, Long, and Serra 2007
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>tur06</code>*
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
Lu et al., 2006
</td>
</tr>
</table>
<p>
Internal loops are not taken into account by the approximative mode.
</p>
<h3>
Long bulge loop effect
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Model</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>san04</code>*
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Missing closing AT penalty.
</td>
<td style="text-align: left;">
SantaLucia et al., 2004
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>tur06</code>*
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
Not tested with experimental results.
</td>
<td style="text-align: left;">
Lu et al., 2006
</td>
</tr>
</table>
<p>
Long bulge loops are not taken into account by the approximative mode.
</p>
<h3>
CNG repeats effect
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Model</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>bro05</code>*
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
Self complementary sequences. 2 to 7 CNG repeats.
</td>
<td style="text-align: left;">
Broda et al., 2005
</td>
</tr>
</table>
<p>
CNG repeats are not taken into account by the approximative mode. The contribution of CNG repeats to the thermodynamic of helix-coil transition can be computed only for 2 to 7 CNG repeats. N represents a single mismatch of type N/N.
</p>
<h3>
Inosine bases effect
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Model</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>san05</code>*
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Missing parameters for tandem base pairs containing inosine bases.
</td>
<td style="text-align: left;">
Watkins Norman E. et al., 2005
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>zno07</code>*
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
Only IU base pairs.
</td>
<td style="text-align: left;">
Wright et al., 2007
</td>
</tr>
</table>
<h3>
Hydroxyadenine bases effect
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Model</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>sug01</code>*
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Only 5’ GA<em>C 3’and 5’ TA</em>A 3’ contexts.
</td>
<td style="text-align: left;">
Kawakami et al., 2001
</td>
</tr>
</table>
<p>
Hydroxyadenine bases (A*) are not taken into account by the approximative mode.
</p>
<h3>
Azobenzenes effect effect
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Model</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>asa05</code>*
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Less reliable results when the number of cis azobenzene increases.
</td>
<td style="text-align: left;">
Asanuma et al., 2005
</td>
</tr>
</table>
<p>
Azobenzenes (X_T for trans azobenzenes and X_C for cis azobenzenes) are not taken into account by the approximative mode.
</p>
<h3>
Locked nucleic acids effect
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Model</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>mct04</code>*
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
McTigue et al., 2004
</td>
</tr>
</table>
<p>
Locked nucleic acids (AL, GL, TL and CL) are not taken into account by the approximative mode.
</p>
<h3>
Sodium corrections
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Correcion</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>ahs01</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Na&gt;0.
</td>
<td style="text-align: left;">
von Ahsen et al., 2001
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>kam71</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Na&gt;0; Na&gt;=0.069; Na&lt;=1.02.
</td>
<td style="text-align: left;">
Frank-Kamenetskii 1971
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>marschdot</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Na&gt;=0.069; Na&lt;=1.02.
</td>
<td style="text-align: left;">
Marmur et al., 1962; Blake et al., 1998
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>owc1904</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Na&gt;0.
</td>
<td style="text-align: left;">
Owczarzy et al., 2004
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>owc2004</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Na&gt;0.
</td>
<td style="text-align: left;">
Owczarzy et al., 2004
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>owc2104</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Na&gt;0.
</td>
<td style="text-align: left;">
Owczarzy et al., 2004
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>owc2204</code>*
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Na&gt;0.
</td>
<td style="text-align: left;">
Owczarzy et al., 2004
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>san96</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Na&gt;=0.1.
</td>
<td style="text-align: left;">
SantaLucia et al., 1996
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>san04</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Na&gt;=0.05; Na&lt;=1.1; Oligonucleotides inferior to 16 bases.
</td>
<td style="text-align: left;">
SantaLucia et al., 2004
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>schlif</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Na&gt;=0.07; Na&lt;=0.12.
</td>
<td style="text-align: left;">
Schildkraut et al., 1965
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>tanna06</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Na&gt;=0.001; Na&lt;=1.
</td>
<td style="text-align: left;">
Tan et al., 2006
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>tanna07</code>*
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
Na&gt;=0.003; Na&lt;=1.
</td>
<td style="text-align: left;">
Tan et al., 2007
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>wet91</code>
</td>
<td style="text-align: left;">
RNA, DNA and RNA/DNA
</td>
<td style="text-align: left;">
Na&gt;0.
</td>
<td style="text-align: left;">
Wetmur 1991
</td>
</tr>
</table>
<h3>
Magnesium corrections
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Correcion</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>oxcmg08</code>*
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Mg&gt;=0.0005; Mg&lt;=0.6.
</td>
<td style="text-align: left;">
Owczarzy et al., 2008
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>tanmg06</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Mg&gt;=0.0001; Mg&lt;=1; Oligomer length superior to 6 base pairs.
</td>
<td style="text-align: left;">
Tan et al., 2006
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>tanmg07</code>*
</td>
<td style="text-align: left;">
RNA
</td>
<td style="text-align: left;">
Mg&gt;=0.1; Mg&lt;=0.3.
</td>
<td style="text-align: left;">
Tan et al., 2007
</td>
</tr>
</table>
<h3>
Mixed Sodium and Magnesium corrections
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Correcion</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>oxcmix08</code>*
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Mg&gt;=0.0005; Mg&lt;=0.6; Na+K+Tris/2&gt;0.
</td>
<td style="text-align: left;">
Owczarzy et al., 2008
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>tanmix07</code>
</td>
<td style="text-align: left;">
DNA and RNA
</td>
<td style="text-align: left;">
Mg&gt;=0.1; Mg&lt;=0.3; Na+K+Tris/2&gt;=0.1; Na+K+Tris/2&lt;=0.3.
</td>
<td style="text-align: left;">
Tan et al., 2007
</td>
</tr>
</table>
<h3>
Sodium equivalent concentration methods
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Correcion</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>ahs01</code>*
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
von Ahsen et al., 2001
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>mit96</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
Mitsuhashi 1996
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>pey00</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
</td>
<td style="text-align: left;">
Peyret 2000
</td>
</tr>
</table>
<h3>
DMSO corrections
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Correcion</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>ahs01</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Not tested with experimental results.
</td>
<td style="text-align: left;">
von Ahsen et al., 2001
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>cul76</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Not tested with experimental results.
</td>
<td style="text-align: left;">
Cullen et al., 1976
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>esc80</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Not tested with experimental results.
</td>
<td style="text-align: left;">
Escara et al., 1980
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>mus80</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
Not tested with experimental results.
</td>
<td style="text-align: left;">
Musielski et al., 1981
</td>
</tr>
</table>
<h3>
Formamide corrections
</h3>
<table summary="Rd table" class="table">
<tr>
<td style="text-align: left;">
<strong>Correcion</strong>
</td>
<td style="text-align: left;">
<strong>Type</strong>
</td>
<td style="text-align: left;">
<strong>Limits/Remarks</strong>
</td>
<td style="text-align: left;">
<strong>Reference</strong>
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>bla96</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
With formamide concentration in mol/L.
</td>
<td style="text-align: left;">
Blake 1996
</td>
</tr>
<tr>
<td style="text-align: left;">
<code>lincorr</code>
</td>
<td style="text-align: left;">
DNA
</td>
<td style="text-align: left;">
With a % of formamide volume.
</td>
<td style="text-align: left;">
McConaughy et al., 1969; Record 1967; Hutton 1977
</td>
</tr>
</table>
<h3>
References
</h3>
<p>
Marmur J and Doty P (1962). “Determination of the base composition of deoxyribonucleic acid from its thermal denaturation temperature.” <em>Journal of Molecular Biology</em>, <b>5</b>(1), pp. 109–118. ISSN 0022-2836, doi: <a href="http://doi.org/10.1016/S0022-2836(62)80066-7">10.1016/S0022-2836(62)80066-7</a>.
</p>
<p>
Schildkraut C and Lifson S (1965). “Dependence of the melting temperature of DNA on salt concentration.” <em>Biopolymers</em>, <b>3</b>(2), pp. 195–208. ISSN 0006-3525, 1097-0282, doi: <a href="http://doi.org/10.1002/bip.360030207">10.1002/bip.360030207</a>, <a href="2018-02-02">2018-02-02</a>.
</p>
<p>
Record MT (1967). “Electrostatic effects on polynucleotide transitions. I. Behavior at neutral pH.” <em>Biopolymers</em>, <b>5</b>(10), pp. 975–992.
</p>
<p>
McConaughy BL, Laird C and McCarthy BJ (1969). “Nucleic acid reassociation in formamide.” <em>Biochemistry</em>, <b>8</b>(8), pp. 3289–3295.
</p>
<p>
Owen R, Hill L and Lapage S (1969). “Determination of DNA base compositions from melting profiles in dilute buffers.” <em>Biopolymers</em>, <b>7</b>(4), pp. 503–516.
</p>
<p>
Frank-Kamenetskii MD (1971). “Simplification of the empirical relationship between melting temperature of DNA, its GC content and concentration of sodium ions in solution.” <em>Biopolymers</em>, <b>10</b>(12), pp. 2623–2624. ISSN 1097-0282, doi: <a href="http://doi.org/10.1002/bip.360101223">10.1002/bip.360101223</a>, <a href="2018-02-02">2018-02-02</a>.
</p>
<p>
Britten RJ, Graham DE and Neufeld BR (1974). “Analysis of repeating DNA sequences by reassociation.” <em>Methods in Enzymology</em>, <b>29</b>, pp. 363–418. doi: <a href="http://doi.org/10.1016/0076-6879(74)29033-5">10.1016/0076-6879(74)29033-5</a>.
</p>
<p>
Cullen BR and Bick MD (1976). “Thermal denaturation of DNA from bromodeoxyuridine substituted cells.” <em>Nucleic acids research</em>, <b>3</b>(1), pp. 49–62.
</p>
<p>
Hutton JR (1977). “Renaturation kinetics and thermal stability of DNA in aqueous solutions of formamide and urea.” <em>Nucleic Acids Research</em>, <b>4</b>(10), pp. 3537–3555. ISSN 0305-1048.
</p>
<p>
Casey J and Davidson N (1977). “Rates of formation and thermal stabilities of RNA:DNA and DNA:DNA duplexes at high concentrations of formamide.” <em>Nucleic Acids Research</em>, <b>4</b>(5), pp. 1539–1552. ISSN 0305-1048.
</p>
<p>
Hall TJ, Grula JW, Davidson EH and Britten RJ (1980). “Evolution of sea urchin non-repetitive DNA.” <em>Journal of Molecular Evolution</em>, <b>16</b>(2), pp. 95–110. ISSN 1432-1432, doi: <a href="http://doi.org/10.1007/BF01731580">10.1007/BF01731580</a>.
</p>
<p>
Escara JF and Hutton JR (1980). “Thermal stability and renaturation of DNA in dimethyl sulfoxide solutions: Acceleration of the renaturation rate.” <em>Biopolymers</em>, <b>19</b>(7), pp. 1315–1327. ISSN 1097-0282, doi: <a href="http://doi.org/10.1002/bip.1980.360190708">10.1002/bip.1980.360190708</a>, <a href="2018-02-02">2018-02-02</a>.
</p>
<p>
Musielski H, Mann W, Laue R and Michel S (1981). “Influence of dimethylsulfoxide on transcription by bacteriophage T3-induced RNA polymerase.” <em>Zeitschrift fur allgemeine Mikrobiologie</em>, <b>21</b>(6), pp. 447–456. ISSN 1521-4028, doi: <a href="http://doi.org/10.1002/jobm.19810210606">10.1002/jobm.19810210606</a>.
</p>
<p>
Freier SM, Kierzek R, Jaeger JA, Sugimoto N, Caruthers MH, Neilson T and Turner DH (1986). “Improved free-energy parameters for predictions of RNA duplex stability.” <em>Proceedings of the National Academy of Sciences</em>, <b>83</b>(24), pp. 9373. <a href="http://www.pnas.org/content/83/24/9373.abstract">http://www.pnas.org/content/83/24/9373.abstract</a>.
</p>
<p>
Breslauer KJ, Frank R, Blocker H and Marky LA (1986). “Predicting DNA duplex stability from the base sequence.” <em>Proceedings of the National Academy of Sciences</em>, <b>83</b>(11), pp. 3746. <a href="http://www.pnas.org/content/83/11/3746.abstract">http://www.pnas.org/content/83/11/3746.abstract</a>.
</p>
<p>
Wahl GM, Barger SL and Kimmel AR (1987). “Molecular hybridization of immobilized nucleic acids: Theoretical concepts and practical considerations.” <em>Methods in Enzymology</em>, <b>152</b>, pp. 399–407.
</p>
<p>
Wetmur JG (1991). “DNA probes: Applications of the principles of nucleic acid hybridization.” <em>Critical Reviews in Biochemistry and Molecular Biology</em>, <b>26</b>(3-4), pp. 227–259. ISSN 1040-9238, doi: <a href="http://doi.org/10.3109/10409239109114069">10.3109/10409239109114069</a>.
</p>
<p>
Chester N and Marshak D (1993). “Dimethyl Sulfoxide-Mediated Primer Tm Reduction: A Method for Analyzing the Role of Renaturation Temperature in the Polymerase Chain Reaction.” <em>Analytical Biochemistry</em>, <b>209</b>(2), pp. 284–290. ISSN 0003-2697, doi: <a href="http://doi.org/10.1006/abio.1993.1121">10.1006/abio.1993.1121</a>.
</p>
<p>
Sugimoto N, Katoh M, Nakano S, Ohmichi T and Sasaki M (1994). “RNA/DNA hybrid duplexes with identical nearest-neighbor base-pairs have identical stability.” <em>FEBS Letters</em>, <b>354</b>(1), pp. 74–78. ISSN 0014-5793, doi: <a href="http://doi.org/10.1016/0014-5793(94)01098-6">10.1016/0014-5793(94)01098-6</a>.
</p>
<p>
Sugimoto N, Nakano S, Katoh M, Matsumura A, Nakamuta H, Ohmichi T, Yoneyama M and Sasaki M (1995). “Thermodynamic Parameters To Predict Stability of RNA/DNA Hybrid Duplexes.” <em>Biochemistry</em>, <b>34</b>(35), pp. 11211–11216. ISSN 0006-2960, doi: <a href="http://doi.org/10.1021/bi00035a029">10.1021/bi00035a029</a>.
</p>
<p>
SantaLucia J, Allawi HT and Seneviratne PA (1996). “Improved Nearest-Neighbor Parameters for Predicting DNA Duplex Stability.” <em>Biochemistry</em>, <b>35</b>(11), pp. 3555–3562. ISSN 0006-2960, doi: <a href="http://doi.org/10.1021/bi951907q">10.1021/bi951907q</a>.
</p>
<p>
Sugimoto N, Nakano S, Yoneyama M and Honda K (1996). “Improved thermodynamic parameters and helix initiation factor to predict stability of DNA duplexes.” <em>Nucleic Acids Research</em>, <b>24</b>(22), pp. 4501–4505. ISSN 0305-1048.
</p>
<p>
Blake RD and Delcourt SG (1996). “Thermodynamic effects of formamide on DNA stability.” <em>Nucleic Acids Research</em>, <b>24</b>(11), pp. 2095–2103. ISSN 0305-1048, doi: <a href="http://doi.org/10.1093/nar/24.11.2095">10.1093/nar/24.11.2095</a>, <a href="2018-02-02">2018-02-02</a>.
</p>
<p>
Blake RD (1996). “Denaturation of DNA.” In Meyers RA (ed.), <em>Encyclopedia of molecular biology and molecular medicine</em>, volume 2, pp. 1–19. VCH Verlagsgesellschaft, Weinheim, Germany.
</p>
<p>
Mitsuhashi M (1996). “Technical report: Part 1. Basic requirements for designing optimal oligonucleotide probe sequences.” <em>Journal of Clinical Laboratory Analysis</em>, <b>10</b>(5), pp. 277–284. ISSN 1098-2825, doi: <a href="http://doi.org/10.1002/(SICI)1098-2825(1996)10:5%3C277::AID-JCLA8%3E3.0.CO;2-5">10.1002/(SICI)1098-2825(1996)10:5&lt;277::AID-JCLA8&gt;3.0.CO;2-5</a>.
</p>
<p>
Allawi HT and SantaLucia J (1997). “Thermodynamics and NMR of Internal G-T Mismatches in DNA.” <em>Biochemistry</em>, <b>36</b>(34), pp. 10581–10594. ISSN 0006-2960, doi: <a href="http://doi.org/10.1021/bi962590c">10.1021/bi962590c</a>, <a href="2018-02-02">2018-02-02</a>.
</p>
<p>
SantaLucia J (1998). “A unified view of polymer, dumbbell, and oligonucleotide DNA nearest-neighbor thermodynamics.” <em>Proceedings of the National Academy of Sciences</em>, <b>95</b>(4), pp. 1460. <a href="http://www.pnas.org/content/95/4/1460.abstract">http://www.pnas.org/content/95/4/1460.abstract</a>.
</p>
<p>
Xia T, SantaLucia J, Burkard ME, Kierzek R, Schroeder SJ, Jiao X, Cox C and Turner DH (1998). “Thermodynamic Parameters for an Expanded Nearest-Neighbor Model for Formation of RNA Duplexes with Watson-Crick Base Pairs.” <em>Biochemistry</em>, <b>37</b>(42), pp. 14719–14735. ISSN 0006-2960, doi: <a href="http://doi.org/10.1021/bi9809425">10.1021/bi9809425</a>.
</p>
<p>
Allawi HT and SantaLucia Jr J (1998). “Thermodynamics of internal C-T mismatches in DNA.” <em>Nucleic Acids Research</em>, <b>26</b>(11), pp. 2694–2701. doi: <a href="http://doi.org/10.1093/nar/26.11.2694">10.1093/nar/26.11.2694</a>.
</p>
<p>
Blake RD and Delcourt SG (1998). “Thermal stability of DNA.” <em>Nucleic Acids Research</em>, <b>26</b>(14), pp. 3323–3332. ISSN 0305-1048.
</p>
<p>
Allawi HT and SantaLucia J (1998). “Nearest Neighbor Thermodynamic Parameters for Internal G-A Mismatches in DNA.” <em>Biochemistry</em>, <b>37</b>(8), pp. 2170–2179. ISSN 0006-2960, doi: <a href="http://doi.org/10.1021/bi9724873">10.1021/bi9724873</a>, <a href="2018-02-02">2018-02-02</a>.
</p>
<p>
Allawi HT and SantaLucia J (1998). “Nearest-Neighbor Thermodynamics of Internal A-C Mismatches in DNA: Sequence Dependence and pH Effects.” <em>Biochemistry</em>, <b>37</b>(26), pp. 9435–9444. ISSN 0006-2960, doi: <a href="http://doi.org/10.1021/bi9803729">10.1021/bi9803729</a>, <a href="2018-02-02">2018-02-02</a>.
</p>
<p>
Mathews DH, Sabina J, Zuker M and Turner DH (1999). “Expanded sequence dependence of thermodynamic parameters improves prediction of RNA secondary structure.” <em>Journal of Molecular Biology</em>, <b>288</b>(5), pp. 911–940. ISSN 0022-2836, doi: <a href="http://doi.org/10.1006/jmbi.1999.2700">10.1006/jmbi.1999.2700</a>.
</p>
<p>
Peyret N, Seneviratne PA, Allawi HT and SantaLucia J (1999). “Nearest-Neighbor Thermodynamics and NMR of DNA Sequences with Internal A-A, C-C, G-G, and T-T Mismatches.” <em>Biochemistry</em>, <b>38</b>(12), pp. 3468–3477. ISSN 0006-2960, doi: <a href="http://doi.org/10.1021/bi9825091">10.1021/bi9825091</a>.
</p>
<p>
Peyret N (2000). <em>Prediction of nucleic acid hybridization: Parameters and algorithms.</em>. PhD thesis, Wayne State University, Detroit, MI.
</p>
<p>
Bommarito S, Peyret N and Jr JS (2000). “Thermodynamic parameters for DNA sequences with dangling ends.” <em>Nucleic Acids Research</em>, <b>28</b>(9), pp. 1929–1934. ISSN 0305-1048.
</p>
<p>
Kawakami J, Kamiya H, Yasuda K, Fujiki H, Kasai H and Sugimoto N (2001). “Thermodynamic stability of base pairs between 2-hydroxyadenine and incoming nucleotides as a determinant of nucleotide incorporation specificity during replication.” <em>Nucleic Acids Research</em>, <b>29</b>(16), pp. 3289–3296. ISSN 0305-1048.
</p>
<p>
von Ahsen N, Wittwer CT and Schutz E (2001). “Oligonucleotide Melting Temperatures under PCR Conditions: Nearest-Neighbor Corrections for Mg2+, Deoxynucleotide Triphosphate, and Dimethyl Sulfoxide Concentrations with Comparison to Alternative Empirical Formulas.” <em>Clinical Chemistry</em>, <b>47</b>(11), pp. 1956–1961. ISSN 0009-9147, 1530-8561, <a href="http://clinchem.aaccjnls.org/content/47/11/1956">http://clinchem.aaccjnls.org/content/47/11/1956</a>.
</p>
<p>
Le Novere N (2001). “MELTING, computing the melting temperature of nucleic acid duplex.” <em>Bioinformatics</em>, <b>17</b>(12), pp. 1226–1227. ISSN 1367-4803, doi: <a href="http://doi.org/10.1093/bioinformatics/17.12.1226">10.1093/bioinformatics/17.12.1226</a>, <a href="2018-02-02">2018-02-02</a>.
</p>
<p>
Ohmichi T, Nakano S, Miyoshi D and Sugimoto N (2002). “Long RNA Dangling End Has Large Energetic Contribution to Duplex Stability.” <em>Journal of the American Chemical Society</em>, <b>124</b>(35), pp. 10367–10372. ISSN 0002-7863, doi: <a href="http://doi.org/10.1021/ja0255406">10.1021/ja0255406</a>.
</p>
<p>
SantaLucia J and Hicks D (2004). “The Thermodynamics of DNA Structural Motifs.” <em>Annual Review of Biophysics and Biomolecular Structure</em>, <b>33</b>(1), pp. 415–440. ISSN 1056-8700, doi: <a href="http://doi.org/10.1146/annurev.biophys.32.110601.141800">10.1146/annurev.biophys.32.110601.141800</a>, <a href="2018-02-02">2018-02-02</a>.
</p>
<p>
Tanaka F, Kameda A, Yamamoto M and Ohuchi A (2004). “Thermodynamic Parameters Based on a Nearest-Neighbor Model for DNA Sequences with a Single-Bulge Loop.” <em>Biochemistry</em>, <b>43</b>(22), pp. 7143–7150. ISSN 0006-2960, doi: <a href="http://doi.org/10.1021/bi036188r">10.1021/bi036188r</a>.
</p>
<p>
McTigue PM, Peterson RJ and Kahn JD (2004). “Sequence-Dependent Thermodynamic Parameters for Locked Nucleic Acid (LNA)-DNA Duplex Formation.” <em>Biochemistry</em>, <b>43</b>(18), pp. 5388–5405. ISSN 0006-2960, doi: <a href="http://doi.org/10.1021/bi035976d">10.1021/bi035976d</a>.
</p>
<p>
Owczarzy R, You Y, Moreira BG, Manthey JA, Huang L, Behlke MA and Walder JA (2004). “Effects of Sodium Ions on DNA Duplex Oligomers: Improved Predictions of Melting Temperatures.” <em>Biochemistry</em>, <b>43</b>(12), pp. 3537–3554. ISSN 0006-2960, doi: <a href="http://doi.org/10.1021/bi034621r">10.1021/bi034621r</a>.
</p>
<p>
Broda M, Kierzek E, Gdaniec Z, Kulinski T and Kierzek R (2005). “Thermodynamic Stability of RNA Structures Formed by CNG Trinucleotide Repeats. Implication for Prediction of RNA Structure.” <em>Biochemistry</em>, <b>44</b>(32), pp. 10873–10882. ISSN 0006-2960, doi: <a href="http://doi.org/10.1021/bi0502339">10.1021/bi0502339</a>.
</p>
<p>
Watkins Norman E. J and SantaLucia John J (2005). “Nearest-neighbor thermodynamics of deoxyinosine pairs in DNA duplexes.” <em>Nucleic Acids Research</em>, <b>33</b>(19), pp. 6258–6267. ISSN 0305-1048, doi: <a href="http://doi.org/10.1093/nar/gki918">10.1093/nar/gki918</a>.
</p>
<p>
Asanuma H, Matsunaga D and Komiyama M (2005). “Clear-cut photo-regulation of the formation and dissociation of the DNA duplex by modified oligonucleotide involving multiple azobenzenes.” <em>Nucleic Acids Symposium Series</em>, pp. 35–36. ISSN 1746-8272, doi: <a href="http://doi.org/10.1093/nass/49.1.35">10.1093/nass/49.1.35</a>.
</p>
<p>
O’Toole AS, Miller S and Serra MJ (2005). “Stability of 3’ double nucleotide overhangs that model the 3’ ends of siRNA.” <em>RNA</em>, <b>11</b>(4), pp. 512–516. ISSN 1355-8382, 1469-9001, doi: <a href="http://doi.org/10.1261/rna.7254905">10.1261/rna.7254905</a>, <a href="2018-03-02">2018-03-02</a>.
</p>
<p>
Lu ZJ, Turner DH and Mathews DH (2006). “A set of nearest neighbor parameters for predicting the enthalpy change of RNA secondary structure formation.” <em>Nucleic Acids Research</em>, <b>34</b>(17), pp. 4912–4924. ISSN 0305-1048, doi: <a href="http://doi.org/10.1093/nar/gkl472">10.1093/nar/gkl472</a>.
</p>
<p>
Kierzek E, Mathews DH, Ciesielska A, Turner DH and Kierzek R (2006). “Nearest neighbor parameters for Watson-Crick complementary heteroduplexes formed between 2’-O-methyl RNA and RNA oligonucleotides.” <em>Nucleic Acids Research</em>, <b>34</b>(13), pp. 3609–3614. ISSN 0305-1048, doi: <a href="http://doi.org/10.1093/nar/gkl232">10.1093/nar/gkl232</a>.
</p>
<p>
Tan Z and Chen S (2006). “Nucleic Acid Helix Stability: Effects of Salt Concentration, Cation Valence and Size, and Chain Length.” <em>Biophysical Journal</em>, <b>90</b>(4), pp. 1175–1190. ISSN 0006-3495, doi: <a href="http://doi.org/10.1529/biophysj.105.070904">10.1529/biophysj.105.070904</a>.
</p>
<p>
O’Toole AS, Miller S, Haines N, Zink MC and Serra MJ (2006). “Comprehensive thermodynamic analysis of 3’ double-nucleotide overhangs neighboring Watson-Crick terminal base pairs.” <em>Nucleic Acids Research</em>, <b>34</b>(11), pp. 3338–3344. ISSN 0305-1048, doi: <a href="http://doi.org/10.1093/nar/gkl428">10.1093/nar/gkl428</a>, <a href="2018-03-02">2018-03-02</a>.
</p>
<p>
Tan Z and Chen S (2007). “RNA Helix Stability in Mixed Na(+)/Mg(2+) Solution.” <em>Biophysical Journal</em>, <b>92</b>(10), pp. 3615–3632. ISSN 0006-3495, doi: <a href="http://doi.org/10.1529/biophysj.106.100388">10.1529/biophysj.106.100388</a>.
</p>
<p>
Wright DJ, Rice JL, Yanker DM and Znosko BM (2007). “Nearest Neighbor Parameters for Inosine-Uridine Pairs in RNA Duplexes.” <em>Biochemistry</em>, <b>46</b>(15), pp. 4625–4634. ISSN 0006-2960, doi: <a href="http://doi.org/10.1021/bi0616910">10.1021/bi0616910</a>.
</p>
<p>
Davis AR and Znosko BM (2007). “Thermodynamic characterization of single mismatches found in naturally occurring RNA.” <em>Biochemistry</em>, <b>46</b>(46), pp. 13425–13436. ISSN 0006-2960, doi: <a href="http://doi.org/10.1021/bi701311c">10.1021/bi701311c</a>.
</p>
<p>
Blose JM, Manni ML, Klapec KA, Stranger-Jones Y, Zyra AC, Sim V, Griffith CA, Long JD and Serra MJ (2007). “Non-Nearest-Neighbor Dependence of Stability for RNA Bulge Loops Based on the Complete Set of Group I Single Nucleotide Bulge Loops.” <em>Biochemistry</em>, <b>46</b>(51), pp. 15123–15135. ISSN 0006-2960, doi: <a href="http://doi.org/10.1021/bi700736f">10.1021/bi700736f</a>.
</p>
<p>
Badhwar J, Karri S, Cass CK, Wunderlich EL and Znosko BM (2007). “Thermodynamic characterization of RNA duplexes containing naturally occurring 1 * 2 nucleotide internal loops.” <em>Biochemistry</em>, <b>46</b>(50), pp. 14715–14724. ISSN 0006-2960, doi: <a href="http://doi.org/10.1021/bi701024w">10.1021/bi701024w</a>, <a href="2018-02-02">2018-02-02</a>.
</p>
<p>
Davis AR and Znosko BM (2008). “Thermodynamic Characterization of Naturally Occurring RNA Single Mismatches with G-U Nearest Neighbors.” <em>Biochemistry</em>, <b>47</b>(38), pp. 10178–10187. ISSN 0006-2960, doi: <a href="http://doi.org/10.1021/bi800471z">10.1021/bi800471z</a>.
</p>
<p>
Miller S, Jones LE, Giovannitti K, Piper D and Serra MJ (2008). “Thermodynamic analysis of 5’ and 3’ single- and 3’ double-nucleotide overhangs neighboring wobble terminal base pairs.” <em>Nucleic Acids Research</em>, <b>36</b>(17), pp. 5652–5659. ISSN 0305-1048, doi: <a href="http://doi.org/10.1093/nar/gkn525">10.1093/nar/gkn525</a>, <a href="2018-03-02">2018-03-02</a>.
</p>
<p>
Owczarzy R, Moreira BG, You Y, Behlke MA and Walder JA (2008). “Predicting Stability of DNA Duplexes in Solutions Containing Magnesium and Monovalent Cations.” <em>Biochemistry</em>, <b>47</b>(19), pp. 5336–5353. ISSN 0006-2960, doi: <a href="http://doi.org/10.1021/bi702363u">10.1021/bi702363u</a>.
</p>
<p>
Dumousseau M, Rodriguez N, Juty N and Novere NL (2012). “MELTING, a flexible platform to predict the melting temperatures of nucleic acids.” <em>BMC Bioinformatics</em>, <b>13</b>, pp. 101. ISSN 1471-2105, doi: <a href="http://doi.org/10.1186/1471-2105-13-101">10.1186/1471-2105-13-101</a>, <a href="2018-02-02">2018-02-02</a>.
</p>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
      </div>

</div>


      <footer><div class="copyright">
  <p>Developed by J. Aravind, Krishna Kumar G..</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="http://pkgdown.r-lib.org/">pkgdown</a>.</p>
</div>

      </footer>
</div>

  </body>
</html>
